flashlib.core.robot/src/main/java/com/flash3388/flashlib/robot/motion/supervised/SupervisedMovable.java

```diff
@@ -0,0 +1,54 @@
+package com.flash3388.flashlib.robot.motion.supervised;
+
+import com.flash3388.flashlib.control.Direction;
+import com.flash3388.flashlib.control.Stoppable;
+import com.flash3388.flashlib.scheduling.Requirement;
+
+public interface SupervisedMovable extends Stoppable, Supervisable, Requirement {
+
+    default void supervisedMove(double speed) {
```

Comment:
It kinda sounds like it should almost be implicit (the idea that the motion is _supervised_), such that a call to `move` is automatically _supervised_.

Remember that it's an interface, you don't actually need to provide all the capabilities in it with `default`, simply require someone to implement it to your specifications.

Same for rest.

Reply:
I forgot to do it... I just copied them and didnt finish it lol